src/main/java/org/candlepin/subscriptions/product/UpstreamProductData.java

```diff
@@ -61,8 +61,7 @@
     SERVICE_TYPE,
     PRODUCT_FAMILY,
     USAGE,
-    /** Name of Offering comes from opProd description field, not the PRODUCT_NAME attribute. */
-    X_DESCRIPTION,
+    PRODUCT_NAME,
```

Comment:
In order to not break in-flight subscription table changes, please add a `description` column to the DB, and populate `description` appropriately here. Then update `SubscriptionCapacityView` to use `o.description` instead of `o.product_name` in its query. (This will ensure the display name of the subscription remains description while still allowing us to use `PRODUCT_NAME` as we wish here).

Reply:
DONE